tests/mock/catalog.rs

```diff
@@ -70,15 +74,15 @@ fn test_catalog_paginate() {
 
     let next = dclient.get_catalog(Some(1));
 
-    let (page1, next) = runtime.block_on(next.into_future()).ok().unwrap();
-    assert_eq!(page1, Some("r1/i1".to_owned()));
+    let (page1, next) = runtime.block_on(next.into_future());
+    assert_eq!(page1.unwrap().unwrap(), "r1/i1".to_owned());
 
-    let (page2, next) = runtime.block_on(next.into_future()).ok().unwrap();
+    let (page2, next) = runtime.block_on(next.into_future());
     // TODO(lucab): implement pagination
-    assert_eq!(page2, None);
+    assert!(page2.is_none());
```

Comment:
This change makes the panic less helpful because it hides the value which was received.

Reply:
Unfortunately, I had to make this due to a limitation of [`error_chain`](https://docs.rs/error-chain/0.12.1/error_chain/):

> Because the error type contains std::error::Error+Send + 'static objects, it can't implement PartialEq for easy comparisons.

The original code results in a compiler error:

```
|     assert_eq!(page2, None);
|     ^^^^^^^^^^^^^^^^^^^^^^^^
|     |
|     std::option::Option<std::result::Result<std::string::String, mock::api_version::dkregistry::errors::Error>>
|     std::option::Option<_>
```

What do you propose to improve the code?